content/chainguard/chainguard-images/overview.md

```diff
@@ -109,3 +109,57 @@ Once you run this command, you'll receive output similar to the following.
 This verifies that the Ruby Chainguard Container is built for both AMD64 and ARM64 architectures.
 
 You can read more about our support of ARM64 in our blog on [Building Wolfi from the ground up](https://www.chainguard.dev/unchained/building-wolfi-from-the-ground-up-and-announcing-arm64-support?utm=docs).
+
+
+## Annotations
```

Comment:
I'd be tempted to call this `Annotations & Labels` as it applies to both and as noted below, they are distinct things with the same values.